FontLoader.js

```diff
@@ -187,36 +249,44 @@
 
 			this._loadFonts();
 		},
+		_cloneAndAppendNode: function(fontFamily, fontFamilyIndex, referenceFontFamiliy) {
+			var clonedDiv = this._testDiv.cloneNode(true);
+			clonedDiv.setAttribute("data-font-family", fontFamily.family);
+			clonedDiv.setAttribute("data-font-family-weight", fontFamily.weight);
+			clonedDiv.setAttribute("data-font-family-style", fontFamily.style);
+			clonedDiv.setAttribute("data-ref-font-family-index", fontFamilyIndex);
+
+			if (referenceFontFamiliy) {
+				clonedDiv.style.fontFamily = referenceFontFamiliy;
+			}
+
+			this._testContainer.appendChild(clonedDiv);
+
+			return clonedDiv;
+		},
+		_setFontStylesForNode: function(node) {
+			node.style.fontFamily = "'" + node.getAttribute("data-font-family") + "', " + FontLoader.referenceFontFamilies[node.getAttribute("data-ref-font-family-index")];
+			node.style.fontWeight = node.getAttribute("data-font-family-weight");
+			node.style.fontStyle = node.getAttribute("data-font-family-style");
+		},
 		_loadFonts: function() {
-			var i, j, clonedDiv, sizeWatcher, sizeWatchers = [],
-				self = this;
```

Comment:
The `self` is used below inside `setTimeout` and `setInterval` functions.
Removing `self` variable breaks the code.